frontend/src/chat/components/basics/QRCode.tsx

```diff
@@ -12,7 +12,12 @@ export const QRCode: React.FC<QRCodeProps> = ({ input, width, height }) => {
   useEffect(() => {
     if (ref.current !== null && input !== '') {
       const codeWriter = new BrowserQRCodeSvgWriter();
-      codeWriter.writeToDom(ref.current, input, width, height);
+      const svg = codeWriter.write(input, width, height);
+      if (ref.current?.firstChild) {
+        ref.current?.replaceChild(svg, ref.current?.firstChild);
+      } else {
+        ref.current?.appendChild(svg);
+      }
```

Comment:
❓  全体的にオプショナルチェーンはなくても大丈夫そうな気がしますが、エラーになりますかね？（個人的にはL13のnullチェックも `ref.current != null` の方が好みですが、まぁ型としては `undefined` は含まれないのでそのままでもいいかな、と）

```suggestion
      const svg = codeWriter.write(input, width, height);
      if (ref.current.firstChild) {
        ref.current.replaceChild(svg, ref.current.firstChild);
      } else {
        ref.current.appendChild(svg);
      }
```

Reply:
オプショナルチェーンがなくてもエラーにならないです。上記のように修正します。